javascript-typing-tutor/styles.css

```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+h1 {
+  font-family: 'Pangolin', cursive;
+  letter-spacing: 3px;
+}
+
+.not-done {
+  color: #f80606;
+  text-decoration: underline;
```

Comment:
The next character to type should be marked with an underline. Even if the user has not tried it yet.